src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetGetOrCreateStorageAccountStep.java

```diff
@@ -43,6 +43,10 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
             DatasetJsonConversion.datasetRequestToDataset(datasetRequestModel),
             profileModel,
             context.getFlightId());
+
+    logger.info("Enable logging");
```

Comment:
Did you mean to leave this log in? If so, I think it could helpful to add more details to the log message, like specifying that it's enabling logging for azure.

Reply:
I did but I can certainly add more meaningful text here